core/src/components/select/select.md.scss

```diff
@@ -142,3 +142,11 @@
 :host(.select-shape-round) {
   --border-radius: 16px;
 }
+
+// Select Native Wrapper
+// ----------------------------------------------------------------
+
+:host(.select-label-placement-stacked) .native-wrapper,
+:host(.select-label-placement-floating) .native-wrapper {
+  width: calc(100% - $select-md-icon-size - $select-icon-margin-start);
```

Comment:
What was the reasoning behind calculating the width instead of using `padding-right` or `margin-right`?